src/include/daos_uns.h

```diff
@@ -52,6 +52,33 @@ struct duns_attr_t {
 	 * This is usually set to false.
 	 */
 	bool			da_no_prefix;
+	/** IN: check only for direct path.
```

Comment:
@mchaarawi the way I am reading the comments it sounds like this is the same as the option above it (da_no_prefix) in that it does not require the daos:// prefix but the addition is that it allows labels?

Reply:
this is to totally bypass the format check.
no_prefix still checks the format of /uuid/uuid/.. or /label/label/.. and parses if it's of that format otherwise goes to check the UNS for the xattrs.
but this new one does not check any xattrs, and just parses as if the direct format was true.
note that is not meant to be used by the data mover or daos tools, and is meant only for MPIIO when users sets an explicit env variable. otherwise it should always be set to false.

Reply:
@mchaarawi ok, I just wanted to make sure I understood the difference between the two fields. For the data mover tools it looks like I will just have to check if the da_cont_label or da_pool_label is set and if it is not then connect/open by uuid instead of label, and I think that is all it needs to work with labels or uuids.